### PR TITLE
Update local demo for service broker and buildpack changes

### PR DESCRIPTION
**Pull request by micahlee**
This PR updates the Conjur CloudFoundry **local** demo to:
- Use the new **supply buildpack** for Conjur/Summon
- Use the new **Org/Space entitlement** features in the service broker
- Use the latest version of Conjur V5 OSS
- Use the latest version of CF Dev

Connected to #23 
Connected to #26 
Connected to #24

Remaining work:
- [x] Verify demo once buildpack and service broker are released
- [x] Update README
- [x] Verify/update demo with Conjur V4
- [x] Update PR description

### CODE REVIEW

File: local/bin/03-install-buildpacks [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -e
+
+get the Conjur buildpack and upload to cf
+mkdir conjur-buildpack
+pushd conjur-buildpack
+  curl -L $(curl -s https://api.github.com/repos/cyberark/cloudfoundry-conjur-buildpack/releases/latest | grep browser_download_url | grep zip | awk '{print $NF}' | sed 's/",*//g') > conjur-buildpack.zip
```
Comment (izgeri) [inline local/bin/03-install-buildpacks]: can we wrap this line so it's not so long?

File: local/bin/03-install-buildpacks [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -e
```
Comment (izgeri) [inline local/bin/03-install-buildpacks]: this should maybe just be `03-install-buildpack` now

File: app/secrets.yml [hunk 3]
```diff
@@ -1,3 +1,9 @@
-DB_USERNAME: !var pcf/app/database/username
-DB_PASSWORD: !var pcf/app/database/password
+ORG_DB_USERNAME: !var pcf/org/database/username
```
Comment (jtuttle) [inline app/secrets.yml]: This file is being copied over from the v4 / v5 versions during the script run, right? We should `.gitignore` and delete it.

File: local/bin/06-privilege-org-space [hunk 4]
```diff
@@ -0,0 +1,36 @@
+#!/bin/bash -e
+ORG_NAME="demo-org"
+SPACE_NAME="demo-space"
+
+# get the host name "pcf/BINDING_ID", and then remove the pcf policy prefix
+# when we load the entitlement directly into the pcf policy,
+# the pcf policy prefix is redundant
+#APP_HOST_NAME=$(cf env $APP_NAME | grep authn_login | awk '{print $NF}' | sed 's/host\///g; s/"//g; s/,$//g')
+#APP_HOST_NAME=$(echo $APP_HOST_NAME | sed 's/pcf\///g')
```
Comment (jtuttle) [inline local/bin/06-privilege-org-space]: Can we get rid of this commented out stuff or is it still useful?

File: local/bin/06-privilege-org-space [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+ORG_GUID=$(cf org --guid "$ORG_NAME")
+SPACE_GUID=$(cf space --guid "$SPACE_NAME")
+
+ORG_LAYER_NAME="$ORG_GUID"
+SPACE_LAYER_NAME="$ORG_GUID/$SPACE_GUID"
+
+echo $ORG_LAYER_NAME
+echo $SPACE_LAYER_NAME
```
Comment (jtuttle) [inline local/bin/06-privilege-org-space]: Should we instead print out something like `"org name: $ORG_LAYER_NAME"`. It'll be pretty obvious what's being printed with the names currently hardcoded at the top of the file, but if those ever change it won't.

**Comment by micahlee** [comment]
Thanks for the great feedback, @jtuttle! This is ready for another look now.

**Pull request merged**